docs/antora/modules/ROOT/pages/setup/docker.adoc

```diff
@@ -211,11 +236,12 @@ make up
 == Stopping Lisk Service
 You can stop Lisk Service again with the following command:
 
+.Working directory: /lisk-service/
 [source,bash]
 ----
 make down
 ----
 
 More commands about how to manage Lisk Service are described on the xref:{url_management}[Docker commands] page.
```

Comment:
Docker commands hasn't been updated, yet. Please comment out the link to the page.